### PR TITLE
Signal readers

### PR DESCRIPTION
**Pull request by tt**

### CODE REVIEW

File: busltee/transport.go [hunk 1]
```diff
@@ -109,28 +111,48 @@ func (t *Transport) Close() error {
...
 	return nil
 }
+
 func (t *Transport) isClosed() bool {
 	t.mutex.Lock()
 	defer t.mutex.Unlock()
 	return t.closed
 }
 
+type nopLocker struct{}
```
Comment (ojacobson) [inline busltee/transport.go]: - [ ] Should this be used somewhere?
- [ ] Should this have a type assertion?

**Comment by ojacobson** [comment]
Basically looks good. I have a couple of Q's (see inline) but I'm happy with the structure.

File: busltee/transport.go (hunk 1 above)
Comment (tt) [inline busltee/transport.go]: It's used on line 34: `sync.NewCond(&nopLocker{})`

The [`sync.Cond`](https://golang.org/pkg/sync/#Cond) type requires a lock but we don't require it as each reader reads from the file.

Comment (ojacobson) [inline busltee/transport.go]: 👍  thanks!

**Pull request merged**